bitnami/spring-cloud-dataflow/templates/server/configmap.yaml

```diff
@@ -51,39 +64,14 @@ data:
           container:
             registry-configurations: {{- include "common.tplvalues.render" (dict "value" .Values.server.configuration.containerRegistries "context" $) | nindent 14 }}
           {{- end }}
-          {{- if .Values.metrics.enabled }}
-          {{- $fullname := include "scdf.fullname" . }}
-          {{- $rsocketPort := int .Values.metrics.service.rsocketPort }}
-          applicationProperties:
-            {{- if .Values.server.configuration.streamingEnabled }}
-            stream:
-              management:
-                metrics:
-                  export:
-                    prometheus:
-                      enabled: true
-                      rsocket:
-                        enabled: true
-                        host: {{ $fullname }}-prometheus-proxy
-                        port: {{ $rsocketPort }}
-            {{- end }}
-            {{- if .Values.server.configuration.batchEnabled }}
-            task:
-              management:
-                metrics:
-                  export:
-                    prometheus:
-                      enabled: true
-                      rsocket:
-                        enabled: true
-                        host: {{ $fullname }}-prometheus-proxy
-                        port: {{ $rsocketPort }}
-            {{- end }}
-          {{- end }}
           {{- if .Values.server.configuration.grafanaInfo }}
-          grafana-info:
+          metrics.dashboard:
             url: {{ .Values.server.configuration.grafanaInfo }}
           {{- end }}
```

Comment:
Wouldn't it be problematic if both `grafanaInfo` and `metricsDashboard` options are set at once? Maybe we could rely on an `if`-`else if` logic instead (and re-ordering the blocks).